di-ipv-credential-issuer-stub/src/main/resources/templates/authorize.mustache

```diff
@@ -667,6 +745,34 @@
         }
       });
     }
+
+    const vcNotBeforeCheckbox = document.getElementById('vcNotBeforeFlg');
+    const vcNotBeforeOverride = document.getElementById('vc_not_before_override');
+
+    if (vcNotBeforeCheckbox) {
+      vcNotBeforeCheckbox.addEventListener('change', function() {
+        if(this.checked) {
+          vcNotBeforeOverride.style.display = 'block';
+        } else {
+          vcNotBeforeOverride.style.display = 'none';
+        }
+      });
+    }
+
+//     set the default value for not before to six months ago
+    const defaultDate = new Date();
+    defaultDate.setMonth(defaultDate.getMonth() - 6);
```

Comment:
In practice we'll be shifting this by a fixed amount (6 * 30 * 24 hours) and I think we should probably match that here.

Trying to shift by months has slightly weird behaviour in edge cases (e.g. this would calculate 6 months before 31st August as either 2nd or 3rd of March, depending on the year 🤔)